.github/labeler.yml

```diff
@@ -0,0 +1,28 @@
+# Use this to test the matches: http://www.globtester.com/
+
+# Any file within the config subfolder
+Config: 'config/**'
+
+# When the .DME is changed
+DME Edit: '**/*.dme'

```

Comment:
Do we really need this tag

Reply:
It's there already, I only added the Github label.